frontend/src/components/tree/TreeGeneralData.tsx

```diff
@@ -52,7 +61,9 @@ const TreeGeneralData: React.FC<TreeGeneralData> = ({ tree }) => {
         ))}
       </dl>
 
-      <section className="mt-16">@TODO: Add image slider</section>
+      <section className="mt-16">
```

Comment:
`<ImageSlider images={images} />`

And put the margin on the wrapper of the slider